lib/css/components/buttons.less

```diff
@@ -140,10 +140,10 @@
     &&__unset {
         &:focus,
         &:focus-visible {
-            outline-style: auto;
+            outline: revert;
         }
 
-        outline: initial;
+        outline: revert;
```

Comment:
`s-btn__unset` already works as I'd expect it to? sorry if that was confusing in my report/codepen 😅 

not sure if this change is to address something else tho. I tried the preview link for the PR and atleast AFAIK tabbing still triggers the same desired visual behavior as before